stix_shifter/src/modules/qradar/json/to_stix_map.json

```diff
@@ -95,7 +95,24 @@
     "key": "url.value"
   },
   "filename": {
-    "key": "file.name"
+    "key": "file.name",
+    "object": "fl"
+  },
+  "sha256hash": {
+    "key": "file.hashes.SHA-256",
+    "object": "fl"
+  },
+  "sha1hash": {
+    "key": "file.hashes.SHA-1",
+    "object": "fl"
+  },
+  "md5hash": {
+    "key": "file.hashes.MD5",
+    "object": "fl"
+  },
+  "filehash": {
+    "key": "file.hashes.UNKNOWN",
```

Comment:
No, and it should never be stored in the STIX bundle as that either. I only have it there because the generic "filehash" column still needs to be captured and parsed into STIX. Basically, this mapping gets overridden when the filehash attribute gets put into STIX.